layouts/_default/baseof.html

```diff
@@ -90,13 +90,7 @@
     {{ hugo.Generator }}
   </head>
 
-  {{ $csClass := "colorscheme-light" }}
-  {{ if eq .Site.Params.colorscheme "dark" }}
-    {{ $csClass = "colorscheme-dark" }}
-  {{ else if eq .Site.Params.colorscheme "auto" }}
-    {{ $csClass = "colorscheme-auto" }}
-  {{ end }}
-  <body class="{{ $csClass }}{{ if .Site.Params.rtl }} rtl{{ end }}"
+  <body class="{{ if .Site.Params.rtl }} rtl{{ end }}"
```

Comment:
I think we still need these color scheme configurations. I want to set a default color theme but let the user change it.
We'll need to run a JS function on load to change the color scheme if the user preference is stored in Local Storage.